package.json

```diff
@@ -24,12 +25,16 @@
     "@babel/preset-env": "^7.22.10",
     "@babel/preset-typescript": "^7.22.5",
     "@types/jest": "^29.5.3",
-    "@walletconnect/types": "^2.9.2",
     "jest": "^29.6.2",
+    "lokijs": "^1.5.12",
```

Comment:
it doesn't look like this is being used in this PR

Reply:
It is needed to run Wallet Connect libraries in NodeJS (to run tests). It is suggested by the WalletConnect package itself. Maybe we can remove this package as it has only added one simple test until we increase the number of tests.